localization/README.md

```diff
@@ -0,0 +1,60 @@
+# Localization
```

Comment:
Would be nice to add a short section to CONTRIBUTING linking to this document.

Reply:
Added in [a7c66d6](https://github.com/blindnet-io/privacy-components-web/pull/118/commits/a7c66d68ff127184df8e65178fa0dda8e36254bb)